internal/reconcile/scc.go

```diff
@@ -27,7 +27,37 @@ func SecurityContextConstraints(k8Client client.Client, desired *security.Securi
 			log.V(3).Info("SecurityContextConstraints are the same skipping update")
 			return nil
 		}
-		return k8Client.Update(context.TODO(), current)
+		return k8Client.Update(context.TODO(), update(desired, current))
 	})
 	return retryErr
 }
+
+func update(from, to *security.SecurityContextConstraints) *security.SecurityContextConstraints {
+	to.Labels = from.Labels
+	to.Priority = from.Priority
+	to.AllowPrivilegedContainer = from.AllowPrivilegedContainer
+	to.DefaultAddCapabilities = from.DefaultAddCapabilities
+	to.RequiredDropCapabilities = from.RequiredDropCapabilities
+	to.AllowedCapabilities = from.AllowedCapabilities
+	to.AllowHostDirVolumePlugin = from.AllowHostDirVolumePlugin
+	to.Volumes = from.Volumes
+	to.AllowedFlexVolumes = from.AllowedFlexVolumes
+	to.AllowHostNetwork = from.AllowHostNetwork
+	to.AllowHostPorts = from.AllowHostPorts
+	to.AllowHostPID = from.AllowHostPID
+	to.AllowHostIPC = from.AllowHostIPC
+	to.DefaultAllowPrivilegeEscalation = from.DefaultAllowPrivilegeEscalation
+	to.AllowPrivilegeEscalation = from.AllowPrivilegeEscalation
+	to.SELinuxContext = from.SELinuxContext
+	to.RunAsUser = from.RunAsUser
+	to.SupplementalGroups = from.SupplementalGroups
+	to.FSGroup = from.FSGroup
+	to.ReadOnlyRootFilesystem = from.ReadOnlyRootFilesystem
+	to.Users = from.Users
+	to.Groups = from.Groups
+	to.SeccompProfiles = from.SeccompProfiles
+	to.AllowedUnsafeSysctls = from.AllowedUnsafeSysctls
+	to.ForbiddenSysctls = from.ForbiddenSysctls
+
```

Comment:
can we replace this block with `*to=*from` if all fields are to be copied?

Reply:
SCCs don't have a "spec" field, they live off the root of the type. The only change we possibly could make is to use reflection over the fields

Reply:
agree. a `*to=*from` would copy more fields than we need.